config/newrelic.yml

```diff
@@ -10,11 +10,11 @@
 
 common: &default_settings
   # Required license key associated with your New Relic account.
-  license_key: ENV["NEWRELIC_LICENSE"]
+  # license_key: ENV["NEW_RELIC_LICENSE_KEY"]
```

Comment:
do we want this commented out?

Reply:
Yes, they are not actually evaluated, as it's in YAML.  I left them there to remind future me that this was where they came from.  Could change the format so it's less confusing, though.

Reply:
Or delete them.  What do you like best?

Reply:
let's keep then but change the formatting so they don't look like commented out keys